packages/css/src/modal.css

```
@@ -1,13 +1,14 @@
 .ds-modal {
   --dsc-modal-backdrop-background: rgb(0 0 0 / 0.5);
   --dsc-modal-background: var(--ds-color-neutral-background-default);
-  --dsc-modal-close-margin: var(--ds-size-3);
+  --dsc-modal-icon-margin: var(--ds-size-3);
```

```suggestion
  --dsc-modal-icon-spacing: var(--ds-size-3);
```
litt usikker på denne - husker ikke helt hvorfor icon margin og icon-top-right-margin er 2 forskjellige 🙈

---

`top-right` må bruke både modal padding og icon margin for å finne ut kor den skal legge seg.